metricbeat/module/elasticsearch/pending_tasks/_meta/data.json

```diff
@@ -0,0 +1,22 @@
+{
+    "@timestamp": "2018-04-02T13:53:10.360Z",
+    "beat": {
+        "name": "host.example.com",
+        "hostname": "host.example.com"
+    },
+    "elasticsearch": {
+      "pending_tasks": {
```

Comment:
Should this be an array?

Reply:
There will be a list of events sent to ES but each single event I think is correct they way it is here.